Implementation_Report_3e/index.html

```diff
@@ -545,7 +545,7 @@ <h2>2. Coding Independent Code Points (CICP) from ITU-R H.273</h2>
                     </ul>
                     <p>Implemented in <a href="https://affinity.serif.com/en-us/photo/"><strong>Affinity Photo 2</strong></a></p>
                     <ul>
-                        <li><a href="https://affinity.help/photo2/en-US.lproj/index.html?page=pages/Appendix/fileformat.html&title=Supported%20file%20formats">Includes 32-bit HDR support (PNG specification - Third edition) for interchanging HDR broadcast imagery in a lossless format. Legacy PNG files exported from Photoshop with CICP data embedded in an ICC profile can be imported.</a></li>
+                        <li><a href="https://affinity.help/photo2/en-US.lproj/index.html?page=pages/Appendix/fileformat.html&title=Supported%20file%20formats">Includes 48-bit HDR support with an Alpha Channel (PNG specification - Third edition) for interchanging HDR broadcast imagery in a lossless format. Legacy PNG files exported from Photoshop with CICP data embedded in an ICC profile can be imported.</a></li>

```

Comment:
No, this is incorrect. Their own announcement (see screenshot in this implementation report) says **32 bits**. Per component!